apps/src/music/views/Timeline.jsx

```diff
@@ -60,7 +54,8 @@ const Timeline = ({isPlaying, currentAudioElapsedTime}) => {
                 <div
                   className={classNames(
                     moduleStyles.barLine,
-                    measure === currentMeasure && moduleStyles.barLineCurrent
+                    measure === Math.floor(currentPlayhead - 1) &&
+                      moduleStyles.barLineCurrent
```

Comment:
super minor nit, but just so we're not overloading the term "measure" here, maybe we could instead save off a local `const measure = index + 1` and use that (so the check is just `measure === Math.floor(currentPlayhead)`)? That way it's clear what actual measure we're rendering here?

Reply:
It's a bit tricky because it's in a `.map` but changed things up to hopefully make it clearer.